components/Navbar.jsx

```diff
@@ -1,5 +1,63 @@
 import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
 
 export const Navbar = () => {
-  return <></>;
+  return (
+    <>
+      <nav>
+        <StyledNavContainer>
```

Comment:
Do you need the `div` container here? Why not put all the styles in the `nav` element?